src/gap.c

```diff
@@ -876,6 +876,59 @@ Obj FuncRUNTIMES( Obj     self)
 }
 
 
+
+/****************************************************************************
+**
+*F  FuncMonotonicNanoseconds( <self> )
+**
+**  'FuncMonotonicNanoseconds' returns an integer which represents the 
+**  number of nanoseconds since some unspecified starting point. This means
+**  that the number returned by this function is not in itself meaningful,
+**  but the difference between the values returned by two consecutive calls
+**  can be used to measure wallclock time.
+**
+**  The accuracy of this is system dependent. For systems that implement
+**  clock_getres, we could get the promised accuracy.
+**
+**  Note that gettimeofday has been marked obsolete in the POSIX standard.
+**  We are using it because it is implemented in most systems still.
+**  Also, if we are using gettimeofday we cannot guarantee the values that
+**  are returned by MonotonicNanoseconds to be monotonic.
```

Comment:
I just wanted to settle the naming bikeshed now. I am inclined to call it
NanosecondsSinceEpoch, since this does what it says on the tin.

In a separate PR I can make another function (or assign a member of GAPInfo?)
that gives properties of the timer, like the resolution and whether its
guaranteed to be monotonic, and document it all.

How does that sound?